app/models/pension/statebenefits/StateBenefitViewModel.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.pension.statebenefits
+
+import models.mongo.TextAndKey
+import play.api.libs.json.{Json, OFormat}
+import utils.DecryptableSyntax.DecryptableOps
+import utils.EncryptableSyntax.EncryptableOps
+import utils.EncryptorInstances.{bigDecimalEncryptor, booleanEncryptor, stringEncryptor}
+import utils.DecryptorInstances.{bigDecimalDecryptor, booleanDecryptor, stringDecryptor}
+import utils.{EncryptedValue, SecureGCMCipher}
+
+case class StateBenefitViewModel(
+                                  benefitId: Option[String] = None,
```

Comment:
They are mandatory on the API so will be populated if retrieved.
But there maybe a case we change it from no to yes.. in which case it will be a new benefit
We can't make it mandatory as we won't have answered that question yet

These are provided for future proofing but may need tweaking when we know more